pkg/artifactory/resource/repository/remote/remote.go

```diff
@@ -53,6 +53,7 @@ type RepositoryBaseParams struct {
 	ContentSynchronisation            *repository.ContentSynchronisation `hcl:"content_synchronisation" json:"contentSynchronisation,omitempty"`
 	MismatchingMimeTypeOverrideList   string                             `hcl:"mismatching_mime_types_override_list" json:"mismatchingMimeTypesOverrideList"`
 	ListRemoteFolderItems             bool                               `json:"listRemoteFolderItems"`
+	DownloadRedirect                  *bool                              `hcl:"download_direct" json:"downloadRedirect,omitempty"`
```

Comment:
We tend to stay away from pointer type 😄 unless there's a reason to use pointer.

There's no need to define `hcl` tag if the key name matches the HCL attribute.

Reply:
I was following the implementation for the local repos here. I can pull the pointer out though. See https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/artifactory/resource/repository/local/local.go#L50.

Reply:
@bmanuel Not your fault 😄 There are still 'legacy' code that works but we haven't had time to move them to where we want, yet. Important thing is not to duplicate/recreate the code that we want to move away from. That's the objective of PR review, right?